java/arcs/core/storage/handle/HandleManager.kt

```diff
@@ -110,7 +112,7 @@ class HandleManager(private val aff: ActivationFactoryFactory? = null) {
             }
         }
 
-        return SetHandle(name, storageProxy, callbacks, refinement).also {
+        return SetHandle(name, storageProxy, callbacks, refinement, canRead).also {
```

Comment:
Possibly switch refinement and canRead around?

Reply:
I kept it this way in case someone called this from somewhere else without using named parameters.